src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GtkMediaPlayer.cs

```diff
@@ -307,10 +307,7 @@ private void GtkMediaPlayer_LayoutUpdated(object? sender, object e)
 				videoWidth = videoTrack.Value.Width;
 				videoHeight = videoTrack.Value.Height;
 
-				if (videoTrack.Value.SarDen != 0)
-				{
-					return true;
-				}
+				return true;
 			}
```

Comment:
@jeromelaban What do you think about this? I'm assuming you added this initially so that `SarDen` being non-zero marks the preparation of a video. This would make sense, but we don't need to wait for SarDen to start layoutting. The details (width and height) can be present even though `SarDen` is not filled out yet. I've failed to pinpoint exactly when or why `SarDen` is filled, but proceeding without `SarDen` temporarily seems to fix the tests (and also works when I've tested manually).

Reply:
Actually, I think the original might be right. This change makes it so that the MPE takes space for the video even though it hasn't loaded yet, filling with black until the playback starts. I will need to think about this further.

Reply:
I recall that checking for SarDen was about waiting for the natural size of the video to be available.